proxy/src/telemetry/metrics/scrape.rs

```diff
@@ -0,0 +1,59 @@
+use std::sync::Arc;
```

Comment:
I'd consider calling this file `prometheus`, since it is specifically going to be in prometheus format, and scrape might be less understandable immediately for Future Us.